LandSandBoat/server
# [SMN] Correct Ifrit animations & Blood Pact abilities

**_I affirm:_**

## What does this pull request do?



Updates all of Ifrit's animations to reflect new pet_skills sql table
Compared all Blood pact msg's to retail captures
Removes Ifrit's animation shifting from core
Added SQL updates for future Ifrit Blood Pact abilities

## Steps to test these changes


Use Ifrit's BP Ward to see proper MP consumption and messages displayed
Use Ifrit's BP Rage to see proper MP consumption and messages displayed
